rust/cross.Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN ./configure
 RUN make install
 RUN ldconfig
 
-RUN curl -o /tmp/openssl.tar.gz https://www.openssl.org/source/old/1.1.1/openssl-1.1.1.tar.gz
+RUN curl -L -o /tmp/openssl.tar.gz https://www.openssl.org/source/old/1.1.1/openssl-1.1.1.tar.gz
```

Comment:
wouldn't curl -Lfo be better to indicate early that it failed to load instead of later when tar tries to extract a html page?